python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -64,8 +64,9 @@ def __init__(self, camera=None, level=None, **kwargs):
         self.level = level
```

Comment:
I can't add this comment on line 61 (as you didn't change that), but please increment the `_VERSION` variable to `1.1` at least.  It would be better to provide backwards compatibility for the new and old versions, but at least catching the version change and having it bomb out with a clear message would be good.  Otherwise, I think old on-disk BFK calibrations will raise.